dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/config_chart_options.tsx

```diff
@@ -29,30 +27,34 @@ export const ConfigChartOptions = ({
 
   const dimensions = useMemo(() => {
     return schemas.map((schema, index) => {
+      let params;
```

Comment:
define params as a blank object here. `{...params}` may fail on undefined. The undefined condition is present, where you are not assigning `params` to anything when `if (schema.eleType === 'colorpicker')` condition fails.